phpstan-baseline.neon

```diff
@@ -378,6 +378,12 @@ parameters:
 			count: 2
 			path: src/ORM/Marshaller.php
 
+		-
+			message: '#^Method Cake\\ORM\\Marshaller\:\:mergeMany\(\) should return array\<TEntity of Cake\\Datasource\\EntityInterface\> but returns list\<Cake\\Datasource\\EntityInterface\>\.$#'
```

Comment:
this is a strange error. Is it saying a template is missing?

Reply:
The template is there though :melting_face: